tests/test_resolver.py

```diff
@@ -289,13 +289,6 @@ async def test_default_loop_for_threaded_resolver(loop: Any) -> None:
     assert resolver._loop is loop
 
 
-@pytest.mark.skipif(aiodns is None, reason="aiodns required")
-async def test_default_loop_for_async_resolver(loop: Any) -> None:
```

Comment:
Why does the test need to be removed?

Reply:
It's only testing the _loop attribute, which I just removed, as it has no real purpose.